examples/simple/WORKSPACE

```diff
@@ -0,0 +1 @@
+# Intentionally blank.
```

Comment:
Do we still need the empty workspace files? In my projects I just have MODULE.bazel now.

Reply:
We need it for `rules_bazel_integration_test`. I added [an issue to track](https://github.com/bazel-contrib/rules_bazel_integration_test/issues/428).